cmd/bootstrap/cmd/finalize.go

```diff
@@ -98,6 +103,18 @@ func addFinalizeCmdFlags() {
 	finalizeCmd.Flags().Uint64Var(&flagNumViewsInStakingAuction, "epoch-staking-phase-length", 100, "length of the epoch staking phase measured in views")
 	finalizeCmd.Flags().Uint64Var(&flagNumViewsInDKGPhase, "epoch-dkg-phase-length", 1000, "length of each DKG phase measured in views")
 	finalizeCmd.Flags().Uint64Var(&flagEpochCommitSafetyThreshold, "epoch-commit-safety-threshold", 500, "defines epoch commitment deadline")
+	// Epoch timing config - these values must be set identically to `EpochTimingConfig` in the FlowEpoch smart contract.
+	// See https://github.com/onflow/flow-core-contracts/blob/240579784e9bb8d97d91d0e3213614e25562c078/contracts/epochs/FlowEpoch.cdc#L259-L266
+	// Must specify either:
+	//   1. --epoch-default-target-end-time and no other `--epoch-target-end-time*` flags
+	//   2. All `--epoch-target-end-time*` flags except --epoch-default-target-end-time
+	//
+	// Use Option 1 for Benchnet, Localnet, etc.
+	// Use Option 2 for Mainnet, Testnet, Canary.
+	finalizeCmd.Flags().BoolVar(&flagDefaultEpochTargetEndTime, "epoch-default-target-end-time", false, "whether to use the default target end time")
+	finalizeCmd.Flags().Uint64Var(&flagEpochTargetEndTimeRefCounter, "epoch-target-end-time-ref-counter", 0, "the reference epoch to use to compute the root epoch's target end time")
```

Comment:
```suggestion
	finalizeCmd.Flags().Uint64Var(&flagRootEpochRefCounter, "root-epoch-ref-counter", 0, "the reference epoch to use to compute the root epoch's target end time")
```

Some suggestion to rename the variable. I found having the "TargetEndTime" in the name is a bit confusing. Having `RootEpoch` is enough. 

Same to the following two flags if you agree this suggestion

Reply:
I changed it to `epoch-timing-ref-counter` instead, since the timing config conceptually defines the timing for all epochs. This also matches the naming of the smart contract config.